framework/test/jest/unit/specs/modules/chain/synchronizer/block_synchronization_mechanism/block_synchronization_mechanism.spec.js

```diff
@@ -14,130 +14,236 @@
 
 'use strict';
 
-const BlockSynchronizationMechanism = require('../../../../../../../../src/modules/chain/synchronizer/block_synchronization_mechanism');
+const {
+	BlockSynchronizationMechanism,
+} = require('../../../../../../../../src/modules/chain/synchronizer/block_synchronization_mechanism');
 const peersList = require('./peers');
+const {
+	Block: blockFixture,
+} = require('../../../../../../../mocha/fixtures/blocks');
 
-describe('BlockSynchronizationMechanism', () => {
-	const stubs = {
-		channel: {
-			invoke: jest.fn(),
-		},
-		modules: {
+describe('block_synchronization_mechanism', () => {
+	afterEach(async () => {
+		jest.clearAllMocks();
+	});
+
+	describe('BlockSynchronizationMechanism', () => {
+		const activeDelegates = 101;
```

Comment:
This shouldn't be hardcoded by instead retrieved from constants/config

Reply:
In jest structure there is no feature to get a compiled configuration and constants. So if you even look at all other tests in the Jest directory you will find hardcoded values everywhere. We may update it but not in scope of this PR.

Reply:
According to @yatki there is a way to overcome this with the current codebase. Could you elaboreate more please @yatki ?

Reply:
@nazarhussain I didn't get why you cannot get the values from config? If getting config function is async, In `before` hook, you can await it, no?

Reply:
Nothing is impossible to do, its just that we don't have that structure available to load configurator and get the compiled configuration. Hope you are working on those utility functions for jest, when we have it we will replace the hardcoded values. Remember this hardcoded value is not only in this test, already used on several other places in jest.

Reply:
Can we have an issue to remember to take care of this then, @nazarhussain ?

Reply:
Here you go https://github.com/LiskHQ/lisk-sdk/issues/4028

Reply:
Thanks!